src/CalcManager/Ratpack/exp.cpp

```diff
@@ -342,11 +342,32 @@ void powratNumeratorDenominator(PRAT *px, PRAT y, uint32_t radix, int32_t precis
         DUPRAT(originalResult, pxPow);
         powratcomp(&originalResult, oneoveryDenom, radix, precision);
                 
+        // ##################################
+        // Find number of digits after decimal point
```

Comment:
Let's update this comment to: Find the decimal place where we should do rounding.